game.js

```diff
@@ -0,0 +1,74 @@
+const prompt = require("prompt-sync")();
+const choices = ['rock', 'paper', 'scissors'];
+
+function ComputerChoice() {
+  let choice = Math.floor(Math.random() * choices.length);
+  return choices[choice];
+}
+
+function PlayerChoice() {
+  let Playerinput = prompt("Enter your choice (Rock, Paper, or Scissors):").toLowerCase();
+
+  if (Playerinput === 'rock') {
+    console.log("You picked rock");
+    return 'rock';
+  }
+  if (Playerinput === 'paper') {
+    console.log("You picked paper");
+    return 'paper';
+  }
+  if (Playerinput === 'scissors') {
+    console.log("You picked scissors");
+    return 'scissors';
+  }
+}
+
+function playRound(playerSelection, computerSelection) {
+  if (playerSelection === computerSelection) {
+    console.log("It's a Tie, Try Again!!!");
+    return "It's a Tie!";
+  } else if (
+    (playerSelection === 'rock' && computerSelection === 'scissors') ||
+    (playerSelection === 'scissors' && computerSelection === 'paper') ||
+    (playerSelection === 'paper' && computerSelection === 'rock')
+  ) {
+    console.log('You Win!');
+    return 'You Win!';
```

Comment:
The expected return is of the form You win, Paper covers Rock, scissors cuts paper or rock smashes scissors